src/vue/components/word-list/word-language-panel.vue

```diff
@@ -118,9 +126,34 @@ export default {
         } else {
           console.warn(`The current filter method - ${this.selectedFilterBy} - is not defined, that's why empty result is returned!`)
         }
-      }
+      }      
       return []
     },
+    wordExactForms () {
+      let exactForms = []
+      this.wordlist.values.map(wordItem => {
+        let exactForm = wordItem.targetWord.toLowerCase()
+        if (exactForms.indexOf(exactForm) === -1) {
+          exactForms.push(exactForm)
+        }
+      })
+      exactForms.sort()
+      return exactForms
+    },
+    wordLemmaForms () {
+      let lemmaForms = []
+      this.wordlist.values.map(wordItem => {
```

Comment:
Probably reduce will be cleaner here, too

Reply:
Fixed